wire-runtime/src/main/java/com/squareup/wire/Message.java

```diff
@@ -166,6 +166,13 @@ public TagMap buildTagMap() {
   }
 
   /** <b>For generated code only.</b> */
+  protected static <T> void redactElements(List<T> list, ProtoAdapter<T> adapter) {
```

Comment:
This is here instead of inlined at the only call site for future code-gen purposes.